runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoOverlay.java

```diff
@@ -176,4 +181,35 @@ else if (opponent instanceof Player)
 
 		return panelComponent.render(graphics);
 	}
+
+	private boolean bosHasHealthbar(int bossId)
```

Comment:
this can be static and the method naming is wrong (should be bossHasHealthBar or something_